src/sphinxcontrib/towncrier/_fragment_discovery.py

```diff
@@ -66,6 +81,39 @@ def lookup_towncrier_fragments(  # noqa: WPS210
         )
         return set()
 
+    fragment_dir = (towncrier_config.directory or 'newsfragments')
```

Comment:
unnecessary parens confuse people into looking for tuples...
```suggestion
    fragment_dir = towncrier_config.directory or 'newsfragments'
```